.typedoc/custom-plugin.mjs

```diff
@@ -1,5 +1,43 @@
 // @ts-check
-import { MarkdownRendererEvent } from 'typedoc-plugin-markdown';
+import { MarkdownPageEvent, MarkdownRendererEvent } from 'typedoc-plugin-markdown';
+
+/**
+ * A list of files where we want to remove any headings
+ */
+const FILES_WITHOUT_HEADINGS = [
+  'use-organization-return.mdx',
+  'use-organization-params.mdx',
+  'paginated-resources.mdx',
+  'pages-or-infinite-options.mdx',
+  'pages-or-infinite-options.mdx',
+  'paginated-hook-config.mdx',
+  'use-organization-list-return.mdx',
+  'use-organization-list-params.mdx',
+];
+
+/**
+ * An array of tuples where the first element is the file name and the second element is the new path.
+ */
+const LINK_REPLACEMENTS = [
```

Comment:
Sometimes there are relative links like `[PaginatedResponse](../types/paginated-response.mdx)` in the files. The link to the also generated file through typedoc. But we already have that page manually created and want to link to it. So this replaces relative links with links to our existing docs.